src/commands/force/lightning/lwc/test/setup.ts

```diff
@@ -129,16 +129,12 @@ export default class Setup extends SfdxCommand {
 
   private installLwcJest(): void {
     this.ux.log('Installing @salesforce/sfdx-lwc-jest node package...');
-    let lwcJestInstallRet;
     const yarnLockExists = fs.existsSync(path.join(this.project.getPath(), 'yarn.lock'));
     if (yarnLockExists) {
       this.ux.log('Detected yarn.lock file, using yarn commands');
-      lwcJestInstallRet = execSync('yarn add --dev @salesforce/sfdx-lwc-jest', { stdio: 'inherit' });
```

Comment:
Are you removing this because the execution is never expected to fail?

Reply:
Before we called `spawnSync` which would return an error prop that we could check and output a pretty error with. This `execSync` call passes through the stdio output to the user so they'll see any success or errors messages from the npm call. 

The call also throws if there's an error, which I think is fine. We could catch and rethrow as a `SfdxError` like we used to, but. I don't think the error message would provide any additional value...

Reply:
If there is any special handling for SfdcError objects, then it would be worth wrapping it.

Reply:
Yeah I guess we can keep the error type just in case and to stay consistent with previous behavior. Added it back.